### ISSUE TITLE
Even unverified creators shows as Brave Verified Creator 

### ISSUE THREAD

**Issue by srirambv** [brave/brave-browser#16570]
## Description 
Even unverified creators shows as Brave Verified Creator 

## Steps to reproduce 

   1. Connect to Uphold account via rewards panel
   2. Visit `https://duckduckgo.com` which is a connected site on staging, shows correct message about `Brave Verified creator`
   3. Visit `https://neowin.net` which is not a connected site on staging, also shows as `Brave Verified creator`

## Actual result 
Connected Sites | Non-Connected Sites
-- | --
![image](https://user-images.githubusercontent.com/17010094/123094684-179d5f00-d425-11eb-8033-18e12b094f54.png) | ![image](https://user-images.githubusercontent.com/17010094/123094703-1ec46d00-d425-11eb-8b6a-8c0403dbd79b.png)

## Expected result
Only show the message for connected sites. 

## Issue reproduces how often 
Easy

## Version/Channel Information:

- Can you reproduce this issue with the current Play Store version? NA
- Can you reproduce this issue with the current Play Store Beta version? Yes 
- Can you reproduce this issue with the current Play Store Nightly version? Yes

## Device details
   - Install type (ARM, x86): ARM
   - Device type (Phone, Tablet, Phablet): Phone
   - Android version: 10
   
## Brave version
1.27.75 x64 Beta Build

### Website problems only
- Does the issue resolve itself when disabling Brave Shields? NA
- Does the issue resolve itself when disabling Brave Rewards? NA
- Is the issue reproducible on the latest version of Chrome? NA

### Additional information

cc: @Miyayes @deeppandya

**Comment by deeppandya** [comment]
@srirambv I think this issue was just to update the text. CC @Miyayes

**Comment by srirambv** [comment]
@deeppandya  yes correct, but if a site has not signed-up to be a Brave Creator it still shows the message as `Brave Verified Creator` for the site which is wrong

**Comment by Miyayes** [comment]
@deeppandya Yeah, so in the neowin example above, it should show this gray message instead: 
![image](https://user-images.githubusercontent.com/11497541/123106780-f168c680-d406-11eb-93a0-3e6ca7883b69.png)

Text is subtly different. (I took this screenshot from desktop.)

**Comment by LaurenWags** [comment]
Unable to reproduce on desktop using STR from description.

```
Brave | 1.27.77 Chromium: 91.0.4472.114 (Official Build) beta (x86_64)
-- | --
Revision | 4bb19460e8d88c3446b360b0df8fd991fee49c0b-refs/branch-heads/4472@{#1496}
OS | macOS Version 10.15.7 (Build 19H1030)
```

This is what the panel shows:
Verified, not KYC'd creator (Connected) | Unverified creator
----- | -----
<img width="396" alt="Screen Shot 2021-06-23 at 1 07 22 PM" src="https://user-images.githubusercontent.com/28145373/123139531-52eb5e00-d424-11eb-9ffe-b3f960ba13d9.png"> | <img width="396" alt="Screen Shot 2021-06-23 at 1 07 33 PM" src="https://user-images.githubusercontent.com/28145373/123139534-541c8b00-d424-11eb-9e05-ba649161c165.png">

This is what the tip banner shows:
Verified, not KYC'd creator (Connected) | Unverified creator
----- | -----
<img width="835" alt="Screen Shot 2021-06-23 at 1 07 55 PM" src="https://user-images.githubusercontent.com/28145373/123139551-5848a880-d424-11eb-967b-8b0cbda84ac4.png"> | <img width="843" alt="Screen Shot 2021-06-23 at 1 07 42 PM" src="https://user-images.githubusercontent.com/28145373/123139557-5aab0280-d424-11eb-9c49-6923079c5ad8.png">

**Comment by jonathansampson** [comment]
This is still an issue with Android instances of Brave.

![image](https://user-images.githubusercontent.com/815158/143898171-017a2976-40cd-4f91-ac31-d0907a1b11da.png)

[Pull-request 10137](https://github.com/brave/brave-core/pull/10137) addressed this issue in part, but [a couple of strings remained unchanged](https://sourcegraph.com/search?q=context:global+repo:%5Egithub%5C.com/brave/brave-core%24+content:%22This+Brave+Verified+Creator%22&patternType=literal):

```xml
<message name="IDS_BRAVE_UI_NOT_VERIFIED_PUBLISHER_DESCRIPTION" desc="A message that appear when a particular publisher is not verified">
  This Brave Verified Creator has not yet configured their account to receive contributions from Brave Users.
  Any tips you send will remain in your wallet until they complete this process.
</message>
```

```xml
<message name="IDS_BRAVE_UI_SITE_BANNER_CONNECTED_TEXT" desc="Text shown for connected publishers that are not fully verified on site banner">
  NOTE: This Brave Verified Creator has not yet configured their account to receive contributions from Brave users. Your browser will keep trying to contribute until they verify, or until 90 days have passed.
</message>
```

```xml
<message name="IDS_BRAVE_UI_PANEL_CONNECTED_TEXT" desc="A message on tipping page when a creator is not verified yet">
  This Brave Verified Creator has not yet configured their account to receive contributions from Brave users. Any tips you send will remain in your wallet until they complete this process.
</message>
```

**Comment by Miyayes** [comment]
This should all be resolved as of 1.35.
